std/java/_std/Type.hx

```diff
@@ -145,7 +145,10 @@ enum ValueType {
 			for (arg in args) {
 				argNum++;
 				var expectedType = argNum < ptypes.length ? ptypes[argNum] : ptypes[ptypes.length - 1]; // varags
-				if (arg == null || expectedType.isAssignableFrom(java.Lib.toNativeType(Type.getClass(arg)))) {
+				var isDynamic = Std.is(arg, DynamicObject) && expectedType.isAssignableFrom(java.Lib.getNativeType(arg));
```

Comment:
Arg may be null, and I think `isAssignableFrom` will crash if the `arg` we send here is null. Try maybe adding something like `var isDynamic = arg == null || (Std.is(arg, DynamicObject) && ...)`

Reply:
Isn't that redundant? `Std.is()` is specified to return `false` in case of `null`.

Reply:
Oh! You are right, of course!